modules/microservice/main.tf

```diff
@@ -536,6 +541,26 @@ resource "azurerm_function_app_slot" "microservice" {
   ]
 }
 
+### Static Site
+  resource "azurerm_storage_account" "microservice" {
+  count = local.has_static_site ? 1 : 0
+
+  name                      = local.environment_differentiator_short_withservice != "" ? "${var.service_name}${var.name}${local.environment_differentiator_short_withservice}${var.environment}" : "${var.service_name}${var.name}${var.environment}"
+  resource_group_name       = var.resource_group_name
+  location                  = var.primary_region
```

Comment:
We will need to support this for all of the regions and get the urls to flow into traffic manager

Reply:
@Matthewsre, I initially placed on the primary region since the storage account can already already multi-region based on its replication type.  (See [docs](https://docs.microsoft.com/en-us/azure/storage/common/storage-redundancy#summary-of-redundancy-options)) .  

However, I can see a possible reason to create multiple copies and place behind traffic manager as you're talking about.  Let me know when you're free to chat.

Reply:
Resolving to pull this in and we will address load balancing multiple regions in the next PR